ipn/ipnlocal/c2n.go

```diff
@@ -319,6 +319,11 @@ func findCmdTailscale() (string, error) {
 			return ts, nil
 		}
 		return "", errors.New("tailscale.exe not found in expected place")
+	case "darwin":
+		if self == "/Applications/Tailscale.app/Contents/MacOS/Tailscale" {
```

Comment:
@agottardo is this reliable? What was that localization issue we saw recently? I forget the details.

Reply:
(but this is at least the common case)

Reply:
If the path is not stable across macOS, I can also check that last element to match `Tailscale` (case-sensitive)

Reply:
I think even `/Contents/MacOS/Tailscale` suffix check should be sufficient?

Reply:
Can you try:

```
mdfind "kMDItemCFBundleIdentifier == 'io.tailscale.ipn.macsys'"
```

It should print the path for the app

Reply:
looking for that issue @bradfitz mentioned